docs/guides/gutenberg-blocks.md

```diff
@@ -10,6 +10,14 @@ _Note: Since the Gutenberg SDK is still being actively developed, the developmen
     yarn docker:wp plugin install gutenberg --activate
     ```
 
+1. In Jetpack, enable loading block assets by adding following filters to your local mu-plugins folder:
```

Comment:
Should we mention somewhere that it will actually "enable" Jetpack blocks?

Reply:
I think "enable loading block assets" is clear enough for now, especially since the filter is meant to be used by a12s mostly, folks working on developing blocks for the SDK.